solana/modules/wormhole-anchor-sdk/src/token_bridge/message.rs

```diff
@@ -63,8 +103,111 @@ impl AnchorDeserialize for TransferWithMeta {
     }
 }
 
-fn to_u16_be(buf: &[u8]) -> u16 {
-    let mut out = [0u8; 2];
-    out.copy_from_slice(buf);
-    u16::from_be_bytes(out)
+impl AnchorSerialize for TransferHeader {
+    fn serialize<W: io::Write>(&self, writer: &mut W) -> io::Result<()> {
+        self.write(writer)
+    }
+}
+
+impl AnchorDeserialize for TransferHeader {
+    fn deserialize_reader<R: io::Read>(reader: &mut R) -> io::Result<Self> {
+        Readable::read(reader)
+    }
+}
+
+/// Bespoke U256 type which as actually a u64 to abstract the padding in (de)serialization.
+#[derive(Debug, PartialEq)]
+struct U256(u64);
```

Comment:
At least the naming seems off, but I think the entire type should likely not exist, because having a separate type for a 3 byte padded u64 seems weird. I can see a more generic type like:
`Padded<const N: usize, T: Readable + Writeable>`  though.